### DISCUSSION TITLE
CoAPP NOT INSTALLING

### DISCUSSION THREAD

**Discussion by bokoisaac** [aclap-dev/video-downloadhelper#1456] [License & Purchase requests]
### What's your issue?

I cannot install the CO APP to verify my license key, it's driving me crazy.

The download is failing.

### The 10 first digits (only 10) of your license number or email address

bokoisaac15@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
hi, could you please provide your versions following this procedure: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F

**Comment by bokoisaac** [comment]
[image: image.png]

On Mon, 11 Nov 2024 at 10:22, jcfrog ***@***.***> wrote:

> hi, could you please provide your versions following this procedure:
> https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/aclap-dev/video-downloadhelper/discussions/1456#discussioncomment-11212438>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5P52VDF4CYQF2H2P6FBOHD2ABZO7AVCNFSM6AAAAABRRA6QR6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMRRGI2DGOA>
> .
> You are receiving this because you authored the thread.Message ID:
> <aclap-dev/video-downloadhelper/repo-discussions/1456/comments/11212438@
> github.com>
>


-- 

*Boko Isaac*
CEO
Marketing | Sibanetics

Phone: 8504005008
Email: ***@***.***
Website: sibanetics.pro

Facebook <https://www.facebook.com/sibaneticsllc> | LinkedIn
<https://www.linkedin.com/in/sibanetics/>

**Comment by bokoisaac** [comment]
hello, I need help??

On Mon, 11 Nov 2024 at 14:50, Boko ***@***.***> wrote:

> [image: image.png]
>
> On Mon, 11 Nov 2024 at 10:22, jcfrog ***@***.***> wrote:
>
>> hi, could you please provide your versions following this procedure:
>> https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/aclap-dev/video-downloadhelper/discussions/1456#discussioncomment-11212438>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/A5P52VDF4CYQF2H2P6FBOHD2ABZO7AVCNFSM6AAAAABRRA6QR6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMRRGI2DGOA>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> <aclap-dev/video-downloadhelper/repo-discussions/1456/comments/11212438@
>> github.com>
>>
>
>
> --
>
> *Boko Isaac*
> CEO
> Marketing | Sibanetics
>
> Phone: 8504005008
> Email: ***@***.***
> Website: sibanetics.pro
>
> Facebook <https://www.facebook.com/sibaneticsllc> | LinkedIn
> <https://www.linkedin.com/in/sibanetics/>
>
>

-- 

*Boko Isaac*
CEO
Marketing | Sibanetics

Phone: 8504005008
Email: ***@***.***
Website: sibanetics.pro

Facebook <https://www.facebook.com/sibaneticsllc> | LinkedIn
<https://www.linkedin.com/in/sibanetics/>

**Reply by jcfrog** [reply]
we did not receive your versions, please use the site, do not reply by email

**Reply by bokoisaac** [reply]
![image](https://github.com/user-attachments/assets/897f3300-8581-46e0-bd08-a92edab28d62)

**Comment by bokoisaac** [comment]
![Screenshot 2024-11-11 144943](https://github.com/user-attachments/assets/d4096e90-9d05-44ab-8ad5-8bffbb98558b)

**Reply by jcfrog** [reply]
can you use the "copy information to clipboard" we can see on your screenshot and then paste it here please?

**Reply by bokoisaac** [reply]
version: 9.1.1.0
target: google
channel: stable
lang: en-US
coapp: {"error":"Specified native messaging host not found.","found":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
{
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 1
}

**Reply by jcfrog** [reply]
OK thanks, 
You say you can't install the coapp, what do you mean, do you get error messages? 

If the installer works try closing all your Chrome windows and reinstall the coapp.

**Comment by bokoisaac** [comment]
Sending a loom...

**Comment by bokoisaac** [comment]
https://www.loom.com/share/c980f65ad6c24dc6928b7ff013c226bc?sid=ac0941c0-5bca-4c5c-94bf-f6c8bfc5ca88

**Comment by jcfrog** [comment]
thanks.
The link seems OK, https://github.com/aclap-dev/vdhcoapp/releases/latest/download/vdhcoapp-windows-x86_64-installer.exe
And it works fine for us. Could it be a region problem or network blocking issue? Do you have a VPN to try downloading from another place?

**Comment by bokoisaac** [comment]
You are right, I just used a VPN and it worked.

Thank youu
